CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+
+# Environnement de développement
+
+## Outils de développement nécessaires
+
+- PHP
+- [composer](https://getcomposer.org)
+- NodeJS
+- [yarn](https://yarnpkg.com)
+- [PostgreSQL](https://www.postgresql.org)
+
+Alternativement, il est possiblement d'utiliser uniquement Docker.
+
+## Installation
+
+    composer install --prefer-dist --no-interaction --no-ansi --no-progress --no-suggest
+    yarn install && yarn build
+
+## Lancement en local
+
+    yarn encore dev --watch
+
+Et dans un autre terminal :
+
+    php bin/console server:run
```

Comment:
`symfony serve -d` using the local web server

Reply:
```
$ symfony serve -d
[InvalidArgumentException]
Command "serve" is not defined.
```

Reply:
It seems you are not using the Symfony CLI here but a PHP tool instead, probably because of an alias in your shell configuration. This should work :) .

Reply:
You're probably right, I may have installed the wrong tool.
Do you have the same output ?
```
$ symfony --version
Symfony Installer version 1.5.11
```

Reply:
I was able to run `symfony serve -d` with the right CLI installed !